apps/settings/js/panels/feedback_send/feedback_send.js

```diff
@@ -136,9 +141,6 @@ define(function(require) {
     },
 
     _responseHandler: function() {
-      if (this._xhr.readyState !== 4) {
-        return;
-      }
       switch (this._xhr.status) {
```

Comment:
HTTP 400 probably means there's a bug in the gaia client or on Input and "please try again" will never work and seems like it'd be unhelpfully frustrating to the user. Maybe log something to the console or somewhere you can see when debugging/testing, but show the user a "success" message?